core/api/utils.py

```diff
@@ -63,10 +63,13 @@ def filter(self, qs, value):
 
 def workbook_response(name, wb):
     """Save xlsx and return the response"""
-    xls = io.BytesIO()
-    wb.save(xls)
-    xls.seek(0)
-    return FileResponse(xls, as_attachment=True, filename=name + ".xlsx")
+    with tempfile.TemporaryDirectory(prefix="mlf-print-") as tmpdirname:
```

Comment:
I would change the prefix name here, to something like `mlf-export-` or something simlar.